.github/workflows/scripts/linux/appimage-qt.sh

```diff
@@ -0,0 +1,251 @@
+	"liblzma.so.5"
+	"libmount.so.1"
+	"libnsl.so.1"
+	"libpcre.so.3"
+	"libselinux.so.1"
+	"libsystemd.so.0"
+	"libudev.so.1"
+	"libwrap.so.0"
+	"libharfbuzz.so.0"
+	"libFLAC.so.8"
+	"libSoundTouch.so.1"
+	"libXau.so.6"
+	"libXcomposite.so.1"
+	"libXcursor.so.1"
+	"libXdamage.so.1"
+	"libXdmcp.so.6"
+	"libXext.so.6"
+	"libXfixes.so.3"
+	"libXi.so.6"
+	"libXinerama.so.1"
+	"libXrandr.so.2"
+	"libXrender.so.1"
+	"libXxf86vm.so.1"
+	"libasyncns.so.0"
+	"libcrypto.so.1.1"
+	"libjpeg.so.8"
+	"liblz4.so.1"
+	"libogg.so.0"
+	"libpcap.so.0.8"
+	"libpng16.so.16"
+	"libpulse.so.0"
+	"libsamplerate.so.0"
+	"libsndfile.so.1"
+	"libvorbis.so.0"
+	"libvorbisenc.so.2"
+	"libxcb.so.1"
+	"libxcb-render.so.0"
+	"libxcb-shm.so.0"
+	"libxkbcommon.so.0"
+	"libxkbcommon-x11.so.0"
+	"pulseaudio/libpulsecommon-13.99.so"
+	"libfreetype.so.6"
+	"libpcre2-16.so.0"
+	"libexpat.so.1"
+	"libffi.so.7"
+	"libgraphite2.so.3"
+	"libresolv.so.2"
+	"libgpg-error.so.0"
+	"libpcre2-16.so.0"
+	"libpng16.so.16"
+	"libxcb-icccm.so.4"
+	"libxcb-image.so.0"
+	"libxcb-keysyms.so.1"
+	"libxcb-randr.so.0"
+	"libxcb-render.so.0"
+	"libxcb-render-util.so.0"
+	"libxcb-shape.so.0"
+	"libxcb-sync.so.1"
+	"libxcb-util.so.1"
+	"libxcb-xfixes.so.0"
+	"libxcb-xkb.so.1"
+	"libevdev.so.2"
+	"libgudev-1.0.so.0"
+	"libinput.so.10"
+	"libjpeg.so.8"
+	"libmtdev.so.1"
+	"libpng16.so.16"
+	"libudev.so.1"
+	"libuuid.so.1"
+)
+
+declare -a DEPLIBS=(
+	"libSDL2-2.0.so.0"
+)
+
+declare -a QTLIBS=(
+	"libQt6Core.so.6"
+	"libQt6Gui.so.6"
+	"libQt6Network.so.6"
+	"libQt6OpenGL.so.6"
+	"libQt6Svg.so.6"
+	"libQt6WaylandClient.so.6"
+	"libQt6WaylandCompositor.so.6"
+	"libQt6WaylandEglClientHwIntegration.so.6"
+	"libQt6WaylandEglCompositorHwIntegration.so.6"
+	"libQt6Widgets.so.6"
+	"libQt6XcbQpa.so.6"
+)
+
+declare -a QTPLUGINS=(
+	"plugins/iconengines"
+	"plugins/imageformats"
+	"plugins/platforms"
+	#"plugins/platformthemes" # Enable this if we want to ship GTK+ themes at any point.
+	"plugins/wayland-decoration-client"
+	"plugins/wayland-graphics-integration-client"
+	"plugins/wayland-graphics-integration-server"
+	"plugins/wayland-shell-integration"
+	"plugins/xcbglintegrations"
+)
+
+set -e
+
+if [ ! -f appimagetool-x86_64.AppImage ]; then
+	wget https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage
+	chmod +x appimagetool-x86_64.AppImage
+fi
+
+OUTDIR=$(realpath "./$APPDIRNAME")
+SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
+rm -fr "$OUTDIR"
+mkdir "$OUTDIR"
+
+mkdir -p "$OUTDIR/usr/bin" "$OUTDIR/usr/lib" "$OUTDIR/usr/lib/pulseaudio"
+
+echo "Copying binary and resources..."
+cp -a "$BINDIR/$BINARY" "$BINDIR/resources" "$BINDIR/shaders" "$OUTDIR/usr/bin"
+
+# Don't need old wx locales.
+rm -fr "$OUTDIR/usr/bin/resources/locale"
+
+# Patch RPATH so the binary goes hunting for shared libraries in the AppDir instead of system.
+echo "Patching RPATH in ${BINARY}..."
+patchelf --set-rpath '$ORIGIN/../lib' "$OUTDIR/usr/bin/$BINARY"
+$STRIP "$OUTDIR/usr/bin/$BINARY"
```

Comment:
Leave the symbols in PCSX2 in please

Being able to debug the appimage, even if it's just function names, is appreciated